bundles/pom.xml

```
@@ -21,6 +21,7 @@
     <module>org.openhab.automation.groovyscripting</module>
     <module>org.openhab.automation.jsscripting</module>
     <module>org.openhab.automation.jythonscripting</module>
+    <module>org.openhab.automation.jrubyscripting</module>
```

Please move the jrubyscripting module up a little higher so the modules remain sorted:

```suggestion
    <module>org.openhab.automation.jrubyscripting</module>
    <module>org.openhab.automation.jsscripting</module>
    <module>org.openhab.automation.jythonscripting</module>
```